src/main/java/com/coveros/selenified/application/Check.java

```diff
@@ -645,15 +637,15 @@ default String checkPromptMatches(String expectedPromptPattern, double waitFor,
      * @param timeTook           - the amount of time it took for wait for something (assuming we had to wait)
      * @return Boolean: whether the cookie is present or not
      */
-    default boolean checkCookieExists(String expectedCookieName, double waitFor, double timeTook) {
+    boolean checkCookieExists(String expectedCookieName, double waitFor, double timeTook) {
         // check for the object to be present
         String cookieValue;
-        boolean isCookiePresent = getApp().is().cookiePresent(expectedCookieName);
+        boolean isCookiePresent = this.app.is().cookiePresent(expectedCookieName);
         if (isCookiePresent) {
-            cookieValue = getApp().get().cookieValue(expectedCookieName);
-            getReporter().pass("to find cookie with the name <b>" + expectedCookieName + STORED, waitFor, COOKIE + expectedCookieName + VALUE_OF + cookieValue + STORED, timeTook);
+            cookieValue = this.app.get().cookieValue(expectedCookieName);
+            this.reporter.pass("to find cookie with the name <b>" + expectedCookieName + STORED, waitFor, COOKIE + expectedCookieName + VALUE_OF + cookieValue + STORED, timeTook);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "to find cookie with the name <b>" 8 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)